versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayPluginUpdatesMojo.java

```diff
@@ -442,7 +398,7 @@ public void execute()
             String effectiveVersion = version;
 
             Artifact artifactRange = getHelper().createPluginArtifact( plugin.getGroupId(), plugin.getArtifactId(),
-                                                                       version );
+                    version );
```

Comment:
I start thinking to introduce spotless as is done in Maven from parent poms 38 when we have such line changed 😄